### ISSUE TITLE
Harder Example Mining

### ISSUE THREAD

**Issue by JamenceTom** [lulujianjie/person-reid-tiny-baseline#21]
Hi! Thanks for your great work!
Is harder example mining a method from a published paper? If yes, can you tell me the name of this paper? Thanks

**Comment by lulujianjie** [comment]
> Hi! Thanks for your great work!
> Is harder example mining a method from a published paper? If yes, can you tell me the name of this paper? Thanks

No, it originates from my project, which is a improved version of hard example mining. The principle of mathematics can be seen in docs [link](https://tiny-reid.readthedocs.io/en/latest/loss/triplet.html)

**Comment by Margrate** [comment]
You add the two lines：
        dist_ap *= (1.0 + self.hard_factor)
        dist_an *= (1.0 - self.hard_factor)
        then, it becomes harder triplet loss?
